root/edit/details/EditWork.js

```diff
@@ -0,0 +1,151 @@
+  const name = display.name;
+  const type = display.type;
+  const languages = display.languages;
+  const attributes = display.attributes;
+  const attributeNames = attributes
+    ? Object.keys(attributes).sort()
+    : null;
+
+  return (
+    <table className="details edit-work">
+      <tr>
+        <th>{addColonText(l('Work'))}</th>
+        <td colSpan="2">
+          <DescriptiveLink entity={display.work} />
+        </td>
+      </tr>
+      {name ? (
+        <WordDiff
+          label={addColonText(l('Name'))}
+          newText={name.new}
+          oldText={name.old}
+        />
+      ) : null}
+      {comment ? (
+        <WordDiff
+          label={addColonText(l('Disambiguation'))}
+          newText={comment.new ?? ''}
+          oldText={comment.old ?? ''}
+        />
+      ) : null}
+      {iswc ? (
+        <Diff
+          label={addColonText(l('ISWC'))}
+          newText={iswc.new ?? ''}
+          oldText={iswc.old ?? ''}
+        />
+      ) : null}
+      {type ? (
+        <FullChangeDiff
+          label={addColonText(l('Work type'))}
+          newContent={
+            type.new?.name ? lp_attributes(type.new.name, 'work_type') : ''
+          }
+          oldContent={
+            type.old?.name ? lp_attributes(type.old.name, 'work_type') : ''
+          }
+        />
+      ) : null}
+      {languages ? (
+        <Diff
+          label={addColonText(l('Lyrics Languages'))}
+          newText={commaOnlyListText(
+            languages.new.map(localizeLanguageName),
+          )}
+          oldText={commaOnlyListText(
+            languages.old.map(localizeLanguageName),
+          )}
+          split=", "
+        />
+      ) : null}
+      {attributeNames ? attributeNames.map((attributeName) => {
+        /*:: if (!attributes) throw 'impossible'; */
+        const attributeDiff = attributes[attributeName];
+        return (
+          <tr key={attributeName}>
+            <th>
+              {addColonText(attributeName)}
+            </th>
+            <td className="old">
+              {attributeDiff.old.length > 0 ? (
+                <ul>
+                  {attributeDiff.old.map((attribute, index) => (
+                    <li key={attribute}>
+                      <DiffSide
+                        filter={DELETE}
+                        newText={attributeDiff.new[index] ?? ''}
+                        oldText={attribute}
+                        split="\s+"
+                      />
+                    </li>
+                  ))}
+                </ul>
+              ) : null}
+            </td>
+            <td className="new">
+              {attributeDiff.new.length > 0 ? (
+                <ul>
+                  {attributeDiff.new.map((attribute, index) => (
+                    <li key={attribute}>
+                      <DiffSide
+                        filter={INSERT}
+                        newText={attribute}
+                        oldText={attributeDiff.old[index] ?? ''}
```

Comment:
This was using `attributeDiff.new` instead of `attributeDiff.old`, so additions weren't being highlighted. Pushed a fix for that